### ISSUE TITLE
Final Project - City Park

### ISSUE THREAD

**Issue by jodicollins** [jodicollins/AndroidBootcamp-CityPark#1]
/cc @codepathreview @codepath - user stories and wireframes are ready for review.

**Comment by nesquena** [comment]
This is very nicely done and very detailed! Great job. Are you working on a team with a few other observers?

**Comment by jodicollins** [comment]
Hi Nathan,

Thanks for the feedback. I'm working on this project alone.

I wanted to followup regarding Project 4 - ExtendedTweets
<https://github.com/jodicollins/AndroidBootcamp-ExtendedTweets>.  I posted
a question in the Issue but it was lengthy.
Basically I am trying to call this from TimeLineActivity.java to get the
HomeTimelineFragment but it is null.

getSupportFragmentManager().
findFragmentByTag("android:switcher:2131230806:0");

When I debugged TweetsPagerAdapter I witnessed the creation of fragments in
this order
android:switcher:2131230806:0
android:switcher:2131230806:1
Any ideas?

Thanks,

Jodi Collins

**Comment by nesquena** [comment]
Use this approach instead http://guides.codepath.com/android/ViewPager-with-FragmentPagerAdapter#access-fragment-instances

**Comment by jodicollins** [comment]
Nathan,

For the final project, when a sprint is complete, is it generally these 3
steps?
1) A push to the existing project in GitHub
2) Raise a new issue
3) Update the README.md, appending to the top of the file.  IE: a new
animated gif and description of work

Best,

Jodi

**Comment by nesquena** [comment]
> For the final project, when a sprint is complete, is it generally these 3
steps?

Yep exactly!

**Comment by jodicollins** [comment]
Nathan,

Do you have a resource for filtering sqlite records based on longitude and latitude? Similar to how Yelp displays the nearest 10 pins to your location. If not, I was planning to try this approach: [Getting Nearest Locations](http://www.drawbackz.com/stack/56480/android-sqlite-getting-nearest-locations-with-latitude-and-longitude.html)

I am using ActiveAndroid and currently have two doubles: longitude and latitude.  There are only 230 records total so I was planning to hit the Parks & Rec API only once and store in sqlite.

Thanks,

Jodi

**Comment by nesquena** [comment]
Check out [Parse Geopoints](https://www.parse.com/docs/android/api/com/parse/ParseQuery.html#whereWithinMiles\(java.lang.String,%20com.parse.ParseGeoPoint,%20double\)). If you use Parse for storing your data this becomes easy. Otherwise, the article you linked to works for local querying.
